docs/docs-content/clusters/cluster-management/cluster-proxy.md

```diff
@@ -0,0 +1,223 @@
+title: "Configure Applications to Use Proxy Server"
+description: "Guide to configure configurations in a cluster to use a proxy server."
+hide_table_of_contents: false
+sidebar_position: 201
+tags: ["clusters", "cluster management", "proxy"]
+---
+
+You can configure applications that operate in your Palette clusters to use a proxy server to access the internet
+instead of accessing the internet directly.
+
+Before you are able to configure applications in a cluster to use the proxy server, you must first ensure that the
+cluster itself is configured to use the proxy server. This means that the host Operating System (OS) and the Palette
+agent inside the cluster will use the proxy server for its outbound communications. The steps to do this vary depending
+on your environment.
+
+After the cluster is configured to use the proxy server, you can proceed to configure the applications inside the
+cluster to use the proxy server. You can do this by applying the `spectrocloud.com/connection: proxy` label to the
+specific job, deployment, or daemon set to instruct an application to use the proxy settings of the cluster.
+
+## Prerequisites
+
+- An active proxy server reachable by your cluster.
+
+<Tabs groupId="scenario">
+
+<TabItem value="Palette SaaS Non-Edge">
+
+- A PCG is deployed into an active and healthy Kubernetes cluster. Refer to
+  [Deploy a PCG to an Existing Kubernetes Cluster](../pcg/deploy-pcg-k8s.md) for additional guidance.
+
+  :::warning
+
+  If you deployed a [PCG through the Palette CLI](../pcg/pcg.md#supported-environments), refer to the respective
+  platform installation guide for instructions on how to configure proxy settings during the installation process
+  through the CLI.
+
+  :::
+
+- The PCG is configured to use the proxy server that you intend for your applications to use for outbound
+  communications. For more information, refer to
+  [Enabled and Manage Proxy Configurations](../pcg/manage-pcg/configure-proxy.md).
+
+</TabItem>
+
+<TabItem value="Self-Hosted Palette Non-Edge">
+
+- A self-hosted Palette instance is deployed into an active and healthy Kubernetes cluster. Refer to
+  [Self-Hosted Palette Installation](../../enterprise-version/install-palette/install-palette.md) for additional
+  guidance.
+
+- The self-hosted Palette instance is configured to use the proxy server that you intend for your applications to use
+  for outbound communications.
+
+</TabItem>
+
+<TabItem value="Edge">
+
+- Your Edge host is configured to use the proxy server that you intend for your applications to use for outbound
+  communications. Refer to [Edge Host Installation](../edge/site-deployment/stage.md) for more information.
+
+</TabItem>
+
+</Tabs>
+
+## Configure Applications to Use Proxy Server
+
+<Tabs groupId="scenario">
+
+<TabItem value="Palette SaaS Non-Edge">
+
+1.  If you are using Palette SaaS, and your cluster does not have direct access to the internet, then the communication
```

Comment:
Clusters do not use PCG to communicate with Palette. The agent inside the cluster communicates directly with Palette through its outbound connection. The PCG reaches out to Palette to receive guidance on what clusters to create or delete. That is all the PCG does. WHen the PCG is instructed to create a cluster, it will provide the configurations required to create the cluster, which will include the proxy settings.